crates/holochain_sqlite/src/conn.rs

```diff
@@ -237,6 +237,7 @@ fn get_encryption_key_shim() -> [u8; 32] {
     ]
 }
 
+// TODO once `conn` has been removed from the public interface, this can be made pub(crate)
```

Comment:
leftover or reserved for future?

Reply:
There are still a few test uses of conn that I think are better tackled after the p2p_* modules are moved out of the holochain_sqlite crate